homeassistant/const.py

```diff
@@ -600,10 +604,25 @@ class Platform(StrEnum):
 IRRADIATION_WATTS_PER_SQUARE_METER: Final = "W/m²"
 IRRADIATION_BTUS_PER_HOUR_SQUARE_FOOT: Final = "BTU/(h×ft²)"
 
+# Precipitation intensity units
+# The derivation of these units is a volume of rain amassing in a container
+# with constant cross section in a given time
+PRECIPITATION_INTENSITY_INCHES_PER_DAY: Final = "in/d"
+PRECIPITATION_INTENSITY_INCHES_PER_HOUR: Final = "in/h"
+PRECIPITATION_INTENSITY_MILLIMETERS_PER_DAY: Final = "mm/d"
+PRECIPITATION_INTENSITY_MILLIMETERS_PER_HOUR: Final = "mm/h"
```

Comment:
There is no such thing as a precipitation unit. I suggest renaming these to their actual unit names instead.

Reply:
There indeed is such a thing. It's explained in the linked architecture issue: https://github.com/home-assistant/architecture/discussions/810

After cancelling out, we get mm/h (for example):
![image](https://user-images.githubusercontent.com/14281572/196780797-af8c026f-8dde-419f-bfb2-1a40fa891953.png)

The problem is that we prefix our unit constants with, in this case `SPEED_` which is misleading when we're measuring the intensity of rainfall.

I can also see how it's not helpful to have multiple constants for "mm/h" though 🤷‍♂️

Reply:
It is expressed as a volumetric flow rate 🤷

Reply:
I've added more context in the architectural discussion here: <https://github.com/home-assistant/architecture/discussions/810#discussioncomment-3918652>